guides/common/modules/ref_supported-clients.adoc

```diff
@@ -0,0 +1,61 @@
+[id="supported-clients_{context}"]
+= Supported Clients
+
+{Project} supports the following operating systems and architectures.
+
+Supported Host Operating Systems::
+The hosts can use the following OS versions:
+
+ifdef::orcharhino[]
+* Alma Linux
+* Amazon Linux
+endif::[]
+ifndef::satellite[]
+* CentOS
+endif::[]
+ifndef::orcharhino,satellite[]
+* CoreOS
```

Comment:
I'm not sure CoreOS is still a thing. Red Hat bought it and now there is [Fedora CoreOS](https://docs.fedoraproject.org/en-US/fedora-coreos/) & RHCOS. https://www.redhat.com/en/technologies/cloud-computing/openshift/what-was-coreos says:

> On May 26, 2020, CoreOS Container Linux reached its end of life and will no longer receive updates.

I'd leave it out.